bigquery/nox.py

```diff
@@ -49,7 +50,8 @@ def system_tests(session, python_version):
 
     # Sanity check: Only run system tests if the environment variable is set.
     if not os.environ.get('GOOGLE_APPLICATION_CREDENTIALS', ''):
-        return
+        raise nox.command.CommandFailed(
+            reason='Credentials must be set via environment variable.')
```

Comment:
This message would be even more helpful if it was a link to the "System Test" section of https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/CONTRIBUTING.rst which describes that variable.